modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/RuleEditor.es.js

```diff
@@ -62,6 +90,7 @@ export const RuleEditor = ({onCancel, onSave, rule, ...otherProps}) => {
 						disabled={disabled}
 						displayType="primary"
 						onClick={() => {
+							convertNumericDataToNumber(localDataStorage.rule);
```

Comment:
By the way, @danvb112, isn't the root cause of this issue the validation that prevents non digits characters, like `,` and `.`, to be added into a Numeric Field?

Perhaps, we should only adjust this validation to support the decimal separators too, what do you think?